api/api.go

```diff
@@ -129,6 +129,8 @@ const (
 	OptCredOwnership = "CredOwnership"
 	// OptCredProxy proxy key in params
 	OptCredProxy = "CredProxy"
+	// OptCredIAMPolicy if set, indicates IAM credentials for s3
```

Comment:
It would be to use IAM for any of the 3 providers

Reply:
Done.